SWIG/randomnumbers.i

```diff
@@ -191,6 +191,18 @@ class SobolRsg {
     const Sample<std::vector<Real> >& nextSequence() const;
     const Sample<std::vector<Real> >& lastSequence() const;
     Size dimension() const;
+    %extend{
+      void skipTo(Size n){
```

Comment:
As far as I see, this doesn't need to go into an `%extend` section (that's a trick that we need when we're hiding smart pointers from the Python interface).  You can simply declare it, like the other methods above.

Reply:
it was needed due to the redefinition (done by me) of the type on that function to boost::uint_least32_t and swig complety doesn't understand what that is. this was the minimal solution